content/en/docs/advanced-tasks/adding-an-admission-webhook.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Shipping an operator that includes Admission Webhooks"
+linkTitle: "Admission Webhooks"
+weight: 3
+---
+
+## Defining your Webhook in the ClusterServiceVersion
+
+### Creating an Admission Webhook
+
+OLM is capable of managing the lifecycle of [validating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) and [mutating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) admission webhooks that are shipped alongside your operator. When developing an [admission webhook](/docs/reference/admission-webhooks) that will be managed by OLM you should consider the following constraints.
+
+#### Certificate Authority Constraints
+
+OLM is configured to provide each deployment with a single Certificate Authority (CA). The logic that generates and mounts the CA into the deployment was originally used by the [API Service](https://kubernetes.io/docs/tasks/access-kubernetes-api/setup-extension-api-server/) lifecycle logic. As a result:
+
+- The tls Cert file will be mounted to the deployment at `/apiserver.local.config/certificates/apiserver.crt`
+- The tls Key file will be mounted to the deployment at `/apiserver.local.config/certificates/apiserver.key`
+
+#### Admission Webhook Rules Constraints
+
+Additionally, in an attempt to prevent operator from configuring the cluster into an unrecoverable state, OLM will place the CSV in the failed phase if the Rules defined in an admission webhook:
+
+- Intercept requests that target all groups
+- Intercept requests that target the `operators.coreos.com` group
+- Intercept requests that target the `ValidatingWebhookConfigurations` or `MutatingWebhookConfigurations` resources
+
+### Defining a Webhook in a CSV
```

Comment:
This section is general and gets to the point. I think this doc will read better if this section is merged with the first section. Maybe something like:

"
OLM is capable of managing the lifecycle of [validating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) and [mutating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) admission webhooks that are shipped alongside your operator. To this end, the [ClusterServiceVersion resource](/docs/Concepts/crds/clusterserviceversion) includes a [WebhookDefinition object](https://github.com/operator-framework/api/blob/master/pkg/operators/v1alpha1/clusterserviceversion_types.go#L164-L180) which can be used to define validating and mutating admission webhooks that will be shipped with the operator. For your convenience, an example of a Validating WebhookDefinition can be seen below:

...

When developing an [admission webhook](/docs/reference/admission-webhooks) that will be managed by OLM you should consider the following constraints:

...
"

Reply:
Addressed in the latest version of the PR.